templates/forms/fields/facebook/facebook.html.twig

```diff
@@ -193,37 +198,48 @@
 
 .social-buttons span i {
   padding-right:4px;
-}</style>
+}
+#fbsummary {
+  display:none;
+}
+</style>
 <script type="text/javascript">
 $('#facebooktitle').keyup(function(){
-  var keyed = $(this).val();$("#facebooktargtitle").html(keyed); 
-  
-});$('#facebookdesc').keyup(function(){var keyed = $(this).val();$("#facebooktargdesc").html(keyed);
-   $("#fbcount").text("Characters left: " + (300 - $(this).val().length));
+  var keyed = $(this).val();$("#facebooktargtitle").html(keyed);
 });
+
+$('#facebookdesc').keyup(function(){
+  var keyed = $(this).val();
+  if (keyed.length > 0) {
+    $("#facebooktargdesc").html(keyed);
+    $("#fbcount").text("Characters left: " + (300 - $(this).val().length));
+  } else {
+    var defaultDesc = $("#toggle_header\\.googledescall1").is(':checked') ? $("#desc").val().substring(0,300) : $("#fbsummary").val();
```

Comment:
Limiting the display of Google description `$("#desc").val().substring(0,300)` to 300 chars